features/org.wso2.developerstudio.eclipse.bps.feature/feature.xml

```diff
@@ -34,6 +34,42 @@
       %license
    </license>
 
+
+	 <requires>
```

Comment:
Fixed in https://github.com/wso2/devstudio-tooling-bps/commit/d915f87d25ae9dd8bedafb49c0c19c1a02fb5c5f